crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -448,13 +482,32 @@ mod tests {
         }
     }
 
+    struct TestBridgeConfig {
+        c8y_prefix: TopicPrefix,
+    }
```

Comment:
Why another struct `TestBridgeConfig` that is exactly a copy of `BridgeConfig`?

Reply:
`BridgeConfig` is part of `C8yMapperConfig` so cannot provide it to operations but one is needed to run tests